ci/docker_run.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Syntax is docker_run.sh IMAGE_NAME SCRIPT_NAME *EXTRA_DOCKER_OPTS
+
+set -x # We want verbosity here, this mostly runs on CI and we want to easily debug stuff
+
+#Note - ensure that the -e flag is NOT set, and explicitly check the $? status to allow for clean up
+
+REMOVE_IMAGE=false
+if [ -z "$branch_specifier" ]; then
+    # manual
+    REMOVE_IMAGE=true
+    IMAGE_NAME="$1"
+else
+    # Jenkins
+    IMAGE_NAME=$branch_specifier"-"$(date +%s%N)
+fi
+
+echo "Running Docker CI build for '$IMAGE_NAME' "
+
+docker build  -t $IMAGE_NAME .
+exit_code=$?; [[ $exit_code != 0 ]] && exit $exit_code
+# Run the command, skip the first argument, which is the local script name
+docker run --sig-proxy=true --rm $IMAGE_NAME ${@:2}
```

Comment:
does this skip the first command , or just execute the second command ? 
e.g.
for these arguments : `a b c d` will this result in `b c d` or just `b` ?

Reply:
Skips the first one, includes the rest. So, `b c d`. The first argument is the docker image name of course

Reply:
I see my comment in the code is incorrect, it says it skips the script name, it actually skips the image name